.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        
+      - name: LaTeX compilation
+        # You may pin to the exact commit or the version.
```

Comment:
That might actually be a good idea. It prevents things from unexpectedly breaking.

Reply:
So to be clear:  could you edit to pin it to the most recent commit that we know works right now?

Reply:
I just updated and pinned the action to version 2.2, which is the same is pinning to a commit, the only difference here is that we are using a git label to label a specific commit (in this case [dacf2cf](https://github.com/xu-cheng/latex-action/commit/dacf2cfbdd5fd768c2298cbdc9e105bd9ed7f293))

Reply:
Fatanstic, thanks! I see that you pinned us to xu-cheng's repository now rather than dante-ve. A superficial scan suggests that's fine, but I'll think about forking it at some point, just to be sure we've always got this around :) In any case, great work :)

Reply:
Yes, i did that because i found out that dante-ve's version was an outdated fork of xu-cheng's repo.